maxun-core/src/interpret.ts

```
@@ -588,37 +592,149 @@ export default class Interpreter extends EventEmitter {
+              }
+            } catch (error) {
+              console.log(`Selector failed: ${selector}`);
+            }
+          }
+
+          if (!workingSelector) {
+            return allResults; 
+          }
+
+          // const nextButton = await page.$(config.pagination.selector);
+          const nextButton = await page.$(workingSelector);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)